StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/RoundedTextField.swift

```diff
@@ -100,7 +101,7 @@ final class RoundedTextField: UIView {
         var theme: ElementsUITheme = .default
         theme.colors = {
             var colors = ElementsUITheme.Color()
-            colors.primary = .brand500
+            colors.primary = self.theme.textFieldFocusedColor
```

Comment:
@mats-stripe 

Is that expected that this changed from `.brand500` to `.brand600`?

Reply:
Whoops - good catch!

Reply:
Fixed in https://github.com/stripe/stripe-ios/pull/3813/commits/cd5a6d409abb53ad71571d277e41bf16de09e4b5!